### DISCUSSION TITLE
Acronyms and Blank page before

### DISCUSSION THREAD

**Discussion by RenataMartins** [joaomlourenco/novathesis#437] [Q&A]
Hello, I am doing my master thesis at UMinho , School of Engineering and I am using version [7.0.2] in overleaf.

I noticed that between the List of Listings and the acronyms there is a blank page, moreover, I am doing my thesis in Portuguese, but the chapter of acronyms is written in English, as in the image.
<img width="290" alt="Acronyms" src="https://github.com/joaomlourenco/novathesis/assets/64305962/bd81f224-721a-4403-b1b3-805160892a96">

Furthermore, in the same way that I have a figure index I have a listing index that doesn't appear in the main index...

Can someone help me?
Thank you in advance.

**Comment by joaomlourenco** [comment]
> I noticed that between the List of Listings and the acronyms there is a blank page

Are you using “listings” or “minted”?

> I am doing my thesis in Portuguese, but the chapter of acronyms is written in English, as in the image.

Confirmed. It is a bug. Will fix asap!

> Furthermore, in the same way that I have a figure index I have a listing index that doesn't appear in the main index...

Are you using “listings” or “minted”?

**Reply by RenataMartins** [reply]
Hi, Thank you for your answer!

1. I'm using listings.

2. Thanks

3. Listings again

**Comment by joaomlourenco** [comment]
> I noticed that between the List of Listings and the acronyms there is a blank page

Can't reproduce the problem.  Can you provide more information (e.g., what are you loading in “4_files.tex”)?

> I am doing my thesis in Portuguese, but the chapter of acronyms is written in English, as in the image.

Fixed in the new version 7.0.3.

> Furthermore, in the same way that I have a figure index I have a listing index that doesn't appear in the main index...

Also fixed in the new version 7.0.3.

**Reply by RenataMartins** [reply]
In 4_file.tex: 
<img width="427" alt="4_files" src="https://github.com/joaomlourenco/novathesis/assets/64305962/9aba4995-33c8-4b34-b60f-8dc58ba92ae1">


In 6_list_of.tex:
<img width="425" alt="6_list_of" src="https://github.com/joaomlourenco/novathesis/assets/64305962/e4d4266d-9ecb-4c87-a833-5f8980951c69">

When I use the "listings" package, a blank page appears before the Acronyms. 
<img width="325" alt="indice_lsitings" src="https://github.com/joaomlourenco/novathesis/assets/64305962/4671d061-bb65-40aa-8c90-83ef007dfed1">
<img width="131" alt="listings" src="https://github.com/joaomlourenco/novathesis/assets/64305962/7250000e-90a4-45c6-8d4b-0b45c57a28c5">


When I use the "minted" package, the listings index appears in English, but the blank page and acronyms disappear.
<img width="364" alt="indice_minted" src="https://github.com/joaomlourenco/novathesis/assets/64305962/ab56bd76-c8c3-4156-a1b5-39f24f3f5cc4">
<img width="133" alt="minted" src="https://github.com/joaomlourenco/novathesis/assets/64305962/27a4ca27-79a6-4401-b17d-9498802ffd28">

**Reply by joaomlourenco** [reply]
I cannot reproduce the bugs you report.
Please **be sure** you are using the latest version from the repository and try again.  If the problem persists, send me a copy of your “source” by email, so that I can see what's happening.

**Reply by RenataMartins** [reply]
Hi, 

I already update to the latest version but i'm still having problems . Could you please give me your email?

**Reply by joaomlourenco** [reply]
My email is "first_name.last_name@fct.unl.pt".

**Reply by joaomlourenco** [reply]
But if you are using overleaf, just share it with me.
